src/pages/Home/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import getToken from '../../lib/tokens';
+
+const Home = () => {
+  const saveToken = () => {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    window.location.reload(false);
+  };
+  return (
+    <main>
+      <h1>Welcome To Your Smart Shopping List</h1>
+      <button onClick={saveToken}>Create a new list</button>
```

Comment:
Alternatively, as April pointed out, one can also put an `event.preventDefault()` in the function that runs onClick to prevent the default submit action. IIRC that might save a reload?

Reply:
Going to use the `event.preventDefault()` solution here.

Reply:
+1 on @anderswift recommendation regarding using `type="button"`.